lib/utils.js

```diff
@@ -3,7 +3,30 @@
  */
 
 export function clone ( source ) {
-  return source ? JSON.parse( JSON.stringify( source ) ) : {};
+  let objCache = [];
+  let cloned = null;
+
+  try {
+    // Clone an object but discard circular references (like document on dom node)
+    // Taken from http://stackoverflow.com/questions/11616630
+    let stringified = JSON.stringify( source, ( key, value ) => {
+      if (typeof value === 'object' && value !== null) {
```

Comment:
Wouldn’t this: `value !== null` break this? https://github.com/kodyl/react-document-meta/issues/4